book/src/dev/rfcs/xxxx-block-subsidy.md

```diff
@@ -47,121 +58,130 @@ In addition to calculations the block subsidy requires constants defined in the
 
 Checking functions for blocks are implemented at `zebra-consensus/src/block/check.rs` and they are called at `zebra-consensus/src/block.rs`. This follows the already existing structure for block validation in Zebra.
 
-It is important to note that the Genesis block, BeforeOverwinter, and Overwinter blocks are verified by the CheckpointVerifier so they are not considered in this design. The following table will show what periods are included and what is not in this proposal:
+It is important to note that the Genesis block, BeforeOverwinter, and Overwinter blocks up to Sapling are verified by the CheckpointVerifier so they are not considered in this design. The following table will show what periods are included and what is not in this proposal:
 
-| Height                                 | Miner    | Founder reward | Funding streams | Shielded Coinbase |
-|----------------------------------------|:--------:|:--------------:|:---------------:|:------------------|             
-| ~Genesis~                              | ~**0%**~ | ~**0%**~       | ~**0%**~        | ~**No**~          |
-| ~Slow Start Shift..Slow Start Interval~| ~80%~    | ~20%~          | ~0%~            | ~No~              |
-| ~Slow Start Interval..Overwinter~      | ~80%~    | ~20%~          | ~0%~            | ~No~              |
-| Overwinter..Sapling                    | 80%      | 20%            | 0%              | No                |
-| Sapling..Blossom                       | 80%      | 20%            | 0%              | No                |
-| Blossom..Heartwood                     | 80%      | 20%            | 0%              | No                |
-| Heartwood..Canopy                      | 80%      | 20%            | 0%              | Yes               |
-| Canopy..Second Halving                 | 80%      | 0%             | 20%             | Yes               |
-| ~Second Halving..~                     | ~100%~   | ~0%~           | ~0%~            | ~Yes~             |
+| Height                                 | Miner    | Founder reward | Funding streams | Shielded Coinbase | Target Spacing |
+|----------------------------------------|----------|----------------|-----------------|-------------------|----------------|             
+| ~Genesis~                              | ~**0%**~ | ~**0%**~       | ~**0%**~        | ~**No**~          | ~**None**~     |
+| ~Slow Start Shift..Slow Start Interval~| ~80%~    | ~20%~          | ~0%~            | ~No~              | ~150 seconds~  |
```

Comment:
We're missing some blocks here:

```suggestion
| ~Genesis~                              | ~0%~ | ~0%~       | ~0%~        | ~No~          | ~None~     |
| ~Genesis..Slow Start Shift~| ~80%~    | ~20%~          | ~0%~            | ~No~              | ~150 seconds~  |
| ~Slow Start Shift..Slow Start Interval~| ~80%~    | ~20%~          | ~0%~            | ~No~              | ~150 seconds~  |
```

Reply:
There's no difference between Genesis..Slow Start Shift and Slow Start Shift..Slow Start Interval in terms of what is in the table. (Slow Start Shift is only relevant in terms of the issuance formula, not the Founders' Reward payment.)

Reply:
Ok, so what we need is just https://github.com/ZcashFoundation/zebra/pull/1129/commits/421f1ea40e591c829e100eea38a6f4eeba98d514
Thanks @daira